engine/gamesys/src/gamesys/scripts/script_resource.cpp

```diff
@@ -1724,6 +1726,9 @@ static int GetAtlas(lua_State* L)
         SET_LUA_TABLE_FIELD(lua_pushstring, "texture", texture_set->m_Texture);
     }
 
+    SET_LUA_TABLE_FIELD(lua_pushnumber, "texture_width", tex_width);
+    SET_LUA_TABLE_FIELD(lua_pushnumber, "texture_height", tex_height);
```

Comment:
We probably want a test case for these. There's a "script_atlas.script" that tests the get_atlas function, perhaps you could add asserts in the "test_get_set_atlas" function in that file? It's located in https://github.com/defold/defold/tree/dev/engine/gamesys/src/gamesys/test/resource

Reply:
@ProtonGustave any chance you could address this concern and add a test?